### PR TITLE
Add instructions to install ruby version 2.4.4

### LINKED ISSUE #5132: Add few more steps to installation instructions
**Description**
Many of us while installing the project faced a common problem of 
```Your Ruby version is 2.x.x, but your Gemfile specified 2.4.4```
Can we add instructions to install the ruby version 2.4.4 using `rvm` or `rbenv` etc before `bundle install` ?

For example: 
`rvm install ruby-2.4.4`
`rvm use ruby-2.4.4`

### PR DESCRIPTION
**Pull request by divyabaid16**
Fixes #5132  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,7 +85,10 @@ For information on how to install for use with the cloud environment, please see
 1. Fork our repo from https://github.com/publiclab/plots2.
 2. In the console, download a copy of your forked repo with `git clone https://github.com/your_username/plots2.git` where `your_username` is your GitHub username.
 3. Enter the new **plots2** directory with `cd plots2`.
-4. Install gems with `bundle install --without production mysql` from the rails root folder, to install the gems you'll need, excluding those needed only in production. You may need to first run `bundle update` if you have older gems in your environment from previous Rails work.
+4. You may need to first run `bundle update` if you have older gems in your environment from previous Rails work. If you get an error message like `Your Ruby version is 2.x.x, but your Gemfile specified 2.4.4` then you need to install the ruby version 2.4.4 using `rvm` or `rbenv`.
+	* Using **rvm**: `rvm install ruby-2.4.4` followed by `rvm use ruby-2.4.4`
+	* Using **rbenv**:  `rbenv install 2.4.4` followed by `rbenv local 2.4.4`
+4. Install gems with `bundle install --without production mysql` from the rails root folder, to install the gems you'll need, excluding those needed only in production.
```
Comment (cesswairimu) [inline README.md]: :+1: Awesome update @divyabaid16  but maybe we can have this as a bullet point `*` because we now have two step `4`. What do you think?

Comment (divyabaid16) [reply README.md]: @cesswairimu I have made the changes.

File: README.md [hunk 2]
```diff
@@ -85,7 +85,11 @@ For information on how to install for use with the cloud environment, please see
 1. Fork our repo from https://github.com/publiclab/plots2.
 2. In the console, download a copy of your forked repo with `git clone https://github.com/your_username/plots2.git` where `your_username` is your GitHub username.
 3. Enter the new **plots2** directory with `cd plots2`.
-4. Install gems with `bundle install --without production mysql` from the rails root folder, to install the gems you'll need, excluding those needed only in production. You may need to first run `bundle update` if you have older gems in your environment from previous Rails work.
+4. Steps to install gems:
+    * You may need to first run `bundle update` if you have older gems in your environment from previous Rails work. If you get an error message like `Your Ruby version is 2.x.x, but your Gemfile specified 2.4.4` then you need to install the ruby version 2.4.4 using `rvm` or `rbenv`.
```
Comment (siaw23) [inline README.md]: I think this line should be removed.
> You may need to first run `bundle update` if you have older gems in your environment from previous Rails work.

 I would argue against running `bundle update`. 
When people clone Plots2 it comes with a `Gemfile.lock` file that has working versions of all the gems we need. Running `bundle update` will update gems in the project that may break or not work. `bundle install` is enough for this.

File: README.md [hunk 3]
```diff
@@ -85,7 +85,11 @@ For information on how to install for use with the cloud environment, please see
...
+	    * Using **rvm**: `rvm install ruby-2.4.4` followed by `rvm use ruby-2.4.4`
```
Comment (siaw23) [inline README.md]: Here's a nice shortcut if you'd like to change to this instead... less typing for (us) the lazy ones 😂...

`rvm install ruby-2.4.4` = `rvm install 2.4.2`
`rvm use ruby-2.4.4` = `rvm use 2.4.4`

File: README.md (hunk 2 above)
Comment (divyabaid16) [reply README.md]: @siaw23 
So if I will use `bundle install` here, then will I need `bundle install --without production mysql` in the next line.
Or should I just make it one point and remove `bundle update` at all?

File: README.md (hunk 3 above)
Comment (divyabaid16) [reply README.md]: @siaw23 I have made the changes.

**Comment by divyabaid16** [comment]
@cesswairimu @gauravano @siaw23 @jywarren Any more changes to be done out here?

File: README.md (hunk 2 above)
Comment (siaw23) [reply README.md]: Replace `bundle update` with `bundle install`. That should be enough.

Comment (divyabaid16) [reply README.md]: I have made the changes.
@siaw23 Thank you so much :smile:

**Comment by gauravano** [comment]
Hey @divyabaid16, could you also update ruby version here - https://github.com/publiclab/plots2/blob/master/doc/PREREQUISITES.md#ruby? Thanks!

**Comment by divyabaid16** [comment]
@gauravano I have updated the file.
Thank you!

**Comment by gauravano** [comment]
Great!

**Comment by jywarren** [comment]
Awesome!!!

**Pull request merged**